IdentityCore/src/telemetry/request_telemetry/MSIDCurrentRequestTelemetry.h

```diff
@@ -26,11 +26,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class MSIDRequestTelemetrySerializedItems;
+
 @interface MSIDCurrentRequestTelemetry : NSObject <MSIDTelemetryStringSerializable>
 
 @property (nonatomic) NSInteger schemaVersion;
 @property (nonatomic) NSInteger apiId;
 @property (nonatomic) BOOL forceRefresh;
+@property (nonatomic) MSIDRequestTelemetrySerializedItems* serializedItems;
```

Comment:
Let's name it "MSIDRequestTelemetrySerializedItem* serializedItem"? "serialized item**s**" confused me, I thought it is array of items. Also, do we need this property to be public? Can we move it to private property?